scripts/bash-comments-lint.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import logging, sys, re, os
+
+LINE_LENGTH = 80
```

Comment:
here, i would have used something like `argparse` to allow the user to change this value, with a default of `80` :yum:

Reply:
absolutely! see [498cadf]()

Reply:
thanks for this :+1: